relayer/relays/beefy/bitfield/bitfield.go

```diff
@@ -0,0 +1,42 @@
+package bitfield
+
+import (
+	"math/big"
+)
+
+type Bitfield []byte
+
+func reverse(thing []byte) {
+	for i, j := 0, len(thing)-1; i < j; i, j = i+1, j-1 {
+		thing[i], thing[j] = thing[j], thing[i]
+	}
+}
+
+// New returns a Bitfield initialized from the Solidity representation of a Bitfield (an array of uint256). See below:
+// https://github.com/Snowfork/snowbridge/blob/18c6225b21782170156729d54a35404d876a2c7b/ethereum/contracts/utils/Bitfield.sol
+func New(input []*big.Int) Bitfield {
+	result := make(Bitfield, 256 * len(input))
```

Comment:
`uint256` is 256 bits wide (32 bytes). This line would allocate 256 bytes per `uint256`. Are we treating each byte as a bit which can only represent 1 or 0?